test/Microsoft.AspNet.Razor.Test/Framework/BlockTypes.cs

```diff
@@ -177,27 +177,43 @@ public MarkupBlock(IEnumerable<SyntaxTreeNode> children)
 
     public class MarkupTagHelperBlock : TagHelperBlock
     {
-        public MarkupTagHelperBlock(string tagName)
-            : this(tagName, new Dictionary<string, SyntaxTreeNode>())
+        public MarkupTagHelperBlock(string tagName, bool selfClosing = false)
+            : this(tagName, new Dictionary<string, SyntaxTreeNode>(), selfClosing)
         {
         }
 
         public MarkupTagHelperBlock(string tagName,
-                                    IDictionary<string, SyntaxTreeNode> attributes)
-            : this(tagName, attributes, new SyntaxTreeNode[0])
+                                    IDictionary<string, SyntaxTreeNode> attributes,
+                                    bool selfClosing = false)
+            : this(tagName, attributes, selfClosing, new SyntaxTreeNode[0])
```

Comment:
- reorder parameters to place `selfClosing` after `tagName`, matching the base class
- add another constructor overload rather than using a default parameter value
- make sure all of these constructor overloads are referenced